packages/terra-site/package.json

```diff
@@ -14,7 +14,8 @@
   },
   "homepage": "https://github.com/cerner/terra-core#readme",
   "scripts": {
-    "start": "webpack-dev-server --progress --port ${PORT:-8080}",
```

Comment:
After the post scrum sync, remove `PORT` for now. Firstly, currently, we have same PORT config. So we can't open webpack-dev-server and run the npm run test at the same time. We may have two different ports for those and also handle the hot reloading of webpack-dev-server.

Secondly, we can use command line to run dev server on any PORT. That will give developers more control on the command arguments.